torch/csrc/distributed/rpc/init.cpp

```diff
@@ -33,6 +33,10 @@ PyObject* rpc_init(PyObject* /* unused */) {
 
   auto module = py::handle(rpc_module).cast<py::module>();
 
+  auto rpcAgentOptions =
```

Comment:
(can be done in a followup PR)

As we are exposing this in API, we should also have docs for it.

Reply:
Tracked this task in #30032.